packages/aws-lambda/src/r5/handler.ts

```diff
@@ -122,6 +122,9 @@ export function fhirSubscriptionHandler(
     if (!event.body) {
       return {
         statusCode: 204,
+        body: JSON.stringify({
+          message: `Body was not provided, noop for the following ${subscription}`,
```

Comment:
Did you test it? My guess is that this print `Unable to find a subscription for [object Object]`.